File: Begginers/Python/dividindo_x_por_y_1116.py
N = int(input())

for c in range(N):
    XY = input().split()
    X = int(XY[0])
    Y = int(XY[1])
    if Y == 0:
       print(f"divisao impossivel")
    else:
       resultado = X / Y
       print(f"{resultado:.1f}")

File: Begginers/Java/dividindo_x_por_y_1116.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
       public static void main(String[] args) throws IOException {
              Locale.setDefault(Locale.US);
              Scanner sc = new Scanner(System.in);

              int N, i;
              double X, Y, divisao;

              N = sc.nextInt();

              for (i = 0; i < N; i++) {
                     X = sc.nextDouble();
                     Y = sc.nextDouble();

                     if (Y == 0) {
                            System.out.println("divisao impossivel");
                     }
                     else {
                            divisao = X / Y;
                            System.out.printf("%.1f%n", divisao);
                     }
              }

              sc.close();

       }
}
